src/test-support/mocks/src/meshes/_/dataplanes/_/_overview.ts

```diff
@@ -74,29 +74,44 @@ export default ({ env, fake }: EndpointDependencies): MockResponder => (req) =>
         ...(isMtlsEnabled ? { mTLS: fake.kuma.dataplaneMtls() } : {}),
         subscriptions: Array.from({ length: subscriptionCount }).map((item, i, arr) => {
           return {
-            id: '118b4d6f-7a98-4172-96d9-85ffb8b20b16',
-            controlPlaneInstanceId: `${fake.hacker.noun()}-${i}`,
+            id: fake.string.uuid(),
+            controlPlaneInstanceId: fake.hacker.noun(),
             ...fake.kuma.connection(item, i, arr),
-            status: {
-              lastUpdateTime: '2021-02-17T10:48:03.638434Z',
-              total: {
-                responsesSent: '5',
-                responsesAcknowledged: '5',
-              },
-              cds: {
-                responsesSent: '1',
-                responsesAcknowledged: '1',
-              },
-              eds: {
-                responsesSent: '2',
-                responsesAcknowledged: '2',
-              },
-              lds: {
-                responsesSent: '2',
-                responsesAcknowledged: '2',
-              },
-              rds: {},
-            },
+            generation: fake.number.int({ min: 1, max: 500 }),
+            status: (() => {
+              const xcks = fake.number.int({ min: 100, max: 500 })
+              const stats = Object.entries(fake.kuma.partitionInto(
+                {
+                  cds: Number,
+                  eds: Number,
+                  lds: Number,
+                  rds: Number,
+                }, xcks,
+              ))
+              return {
+                lastUpdateTime: '2021-07-13T09:03:11.614941842Z',
```

Comment:
What about using a random date-time?
```suggestion
                lastUpdateTime: fake.date.past().toISOString(),
```

Reply:
I don't think this gives us the exact format for the dates this seems to use. I think the backend gives us dates to nano second precision. Ideally we'd use https://github.com/kumahq/kuma-gui/pull/3158 here for that.

Lemme see if I can get that other PR merged in and we can circle back on these all in one go.